mlflow/pipelines/cards/pandas_renderer.py

```diff
@@ -136,11 +136,17 @@ def convert_to_dataset_feature_statistics(
                 if equal_height_hist:
                     feat_stats.histograms.append(equal_height_hist)
         elif feat.type == fs_proto.STRING:
+            isCurrentColumnBooleanType = False
+            if current_column_value.dtype == bool:
+                current_column_value = current_column_value.replace({True: "True", False: "False"})
+                isCurrentColumnBooleanType = True
```

Comment:
Instead of doing string manipulation here, it feels more conventional to treat boolean as integers. You can probably do that on the dataframe level, similar to what's discussed here: https://stackoverflow.com/questions/17383094/how-can-i-map-true-false-to-1-0-in-a-pandas-dataframe

Reply:
Spoke offline 
```
Booleans are a categorical feature. If you convert them to int then you would also end up computing avg, min, max, … which do not make sense in this context.
I would also say that this is an anti-pattern — usually we try to identify ints that represent catagorical features (e.g., day_of_week). I am not sure we want to go in the opposite direction 
```